mutable_tree.go

```diff
@@ -827,6 +827,11 @@ func (tree *MutableTree) SaveVersion() ([]byte, int64, error) {
 	if version == 1 && tree.ndb.opts.InitialVersion > 0 {
 		version = int64(tree.ndb.opts.InitialVersion)
 	}
+	if tree.ndb.keysAccessed != nil {
```

Comment:
I think replacing entire set with new one might be better than removing all keys one-by-one.

Reply:
For sure. If it doesn't leak any memory, that works.